docs/full-node-api/overview.md

```diff
@@ -2,37 +2,86 @@
 title: "Overview"
 ---
 
-# Overview
+# Full Node Setup Guide for Bitfinity Network
 
-The EVM has support for a limited number of blocks. To access all the historical data, you can run a full node locally, called the block-importer. 
+## Introduction to Full Nodes
 
-Bitfinity currently supports a read-only full-node implementation to access the data on the EVM. Our Full node is a fork of Erigon that enables a user to download historical blocks, executes all transactions and validates the state roots. In addition, the EVM is capable of signing blocks, and the block-importer can validate the signatures to ensure the blcoks were indeed generated by the EVM.  
+### What is a Full Node?
 
-We will shortly be upgrading the full node, to enable broadcasting transactions. 
+A full node plays a pivotal role in blockchain networks by validating transactions and blocks in accordance with the network's consensus rules. It contributes to the decentralization and security of the network, offering users a private and trustless interface to the blockchain.
 
-## Build and Run a Full Node
+## Bitfinity Full Node Overview
 
-To sync with the node you will need to build the code by following the instructions in https://github.com/bitfinity-network/erigon/blob/evmc_importer/BITFINITY.README.md on the evmc_importer branch. 
```

Comment:
Actually block-importer only re-executes a block. Another important part is rpcdaemon that processes JSON RPC requests. Both together provide similar to full node functionality

Reply:
This part has been removed